apps/frontend/src/utils/recaptcha/load-google-recaptcha.ts

```diff
@@ -0,0 +1,18 @@
+const reCaptchaSiteKey = process.env.NEXT_PUBLIC_RECAPTCHA_SITE_KEY;
+
+export default async function loadGoogleRecaptcha(): Promise<void> {
```

Comment:
I'd rename this to `ensureGoogleRecaptchaLoaded` since this function does not load Recaptcha every time.